docs/resources/secmaster_indicator.md

```diff
@@ -0,0 +1,123 @@
+---
+subcategory: "SecMaster"
+---
+
+# huaweicloud_secmaster_indicator
+
+Manages a SecMaster indicator resource within HuaweiCloud.
+
+~> This resource can only be used in region **cn-east-3** for now.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+
+resource "huaweicloud_secmaster_indicator" "test" {
+  workspace_id = var.workspace_id
+  name         = "test_update"
+  
+  type {
+    category       = "Domain"
+    indicator_type = "Domain"
+  }
+
+  data_source {
+    source_type     = "1"
+    product_feature = "hss"
+    product_name    = "hss"
+  }
+
+  status                = "Open"
+  confidence            = 80
+  first_occurrence_time = "2023-10-24T17:23:55.000+08:00"
+  last_occurrence_time  = "2023-10-25T11:15:30.000+08:00"
+  threat_degree         = "Black"
+  granularity           = "1"
+  value                 = "test.terraform.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the ID of the workspace to which the indicator belongs.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the indicator name.
+
+* `type` - (Required, List) Specifies the indicator type.
+  The [type](#Indicator_IndicatorType) structure is documented below.
+
+* `threat_degree` - (Required, String) Specifies the threat degree.
+  The value can be: **Black**, **White** and **Gray**.
+
+* `data_source` - (Required, List, ForceNew) Specifies the data source of the indicator.
+  The [data_source](#Indicator_DataSource) structure is documented below.
+
+  Changing this parameter will create a new resource.
+
+* `status` - (Required, String) Specifies the indicator status.
+  The value can be: **Open**, **Closed** and **Revoked**.
+
+* `confidence` - (Required, Int) Specifies the confidence of the indicator.
+  The value ranges from **80** to **100**.
+
+* `first_occurrence_time` - (Required, String) Specifies the first occurrence time of the indicator.
+
+* `last_occurrence_time` - (Required, String) Specifies the last occurrence time of the indicator.

```

Comment:
can we use the time which is earlier than the current time?

Reply:
no limit, any time is OK